src/main/java/com/microsoft/aad/msal4j/HttpHelper.java

```diff
@@ -6,32 +6,63 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
+import static com.microsoft.aad.msal4j.Constants.POINT_DELIMITER;
 
 class HttpHelper {
 
     private final static Logger log = LoggerFactory.getLogger(HttpHelper.class);
+    public static final String RETRY_AFTER_HEADER = "Retry-After";
+    public static final int RETRY_NUM = 2;
+    public static final int RETRY_DELAY_MS = 1000;
+
+    private static String getRequestThumbprint(RequestContext requestContext) {
```

Comment:
Is this method supposed to work on other request types than SilentRequest? I think the thumbprints for non-silent requests are not valid, as they contain too little data. Consider renaming to `GetSilentRequestThumbprint`..

Reply:
Under requests we mean set of parameters passed to MSAL api - it is applicable to any request